openedx/core/djangoapps/user_api/legacy_urls.py

```diff
@@ -0,0 +1,41 @@
+"""
+Defines the URL routes for this app.
+"""
+
+from django.conf import settings
+from django.conf.urls import include, patterns, url
+from rest_framework import routers
+
+from . import views as user_api_views
+from .models import UserPreference
+
+
```

Comment:
I'm confused about the newline rule. I thought that two lines were needed after the imports. I guess it is instead that two lines are needed before each class.